Fix CI test flakiness

# Goal

CI tests should run reliably. Currently, the `test` job fails unpredictably. Investigate cause and fix.

## Details

The `test` job fails (sometimes) with the following error:
```
> frontend@0.0.1 test:integration
> npx playwright test --config tests/integration
[WebServer]
...
Error: Timed out waiting 60000ms from config.webServer.
Error: Error: Timed out waiting 60000ms from config.webServer.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/javascript.yml
+++ b/.github/workflows/javascript.yml
@@ -46,8 +46,12 @@ jobs:
         run: bash scripts/build-deps.sh
       - name: Install Playwright
         run: npx playwright install chromium --with-deps
+      # use separate build step to avoid playwright timeout issue
+      - name: Build SvelteKit vite app
+        run: npm run build
+      # skip build in test step
       - name: Run Tests
-        run: npm run test
+        run: npm run test --skip-build
 
   build:
     runs-on: ubuntu-latest

--- a/src/lib/wallet/WrongNetwork.svelte
+++ b/src/lib/wallet/WrongNetwork.svelte
@@ -2,6 +2,11 @@
 	import { switchNetwork } from '@wagmi/core';
 	import { AlertItem, AlertList, Button } from '$lib/components';
 
+	// NOTE: don't move this to inline handler (prevent vite build warning)
+	function handleClick() {
+		switchNetwork({ chainId });
+	}
+
 	export let chainId: number;
 	export let chainName: string;
 	export let size: 'xs' | 'sm' | 'md' | 'lg' = 'md';
@@ -18,7 +23,7 @@
 	<AlertList {size} status="error">
 		<AlertItem title="Wrong network">
 			Please connect to {chainName}
-			<Button slot="cta" size={buttonSize[size]} label="Switch network" on:click={() => switchNetwork({ chainId })} />
+			<Button slot="cta" size={buttonSize[size]} label="Switch network" on:click={handleClick} />
 		</AlertItem>
 	</AlertList>
 </div>

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -4,9 +4,19 @@
  * https://vitejs.dev/config/
  */
 import { sveltekit } from '@sveltejs/kit/vite';
+import { createLogger } from 'vite';
 import { defineConfig } from 'vitest/config';
 import jsonServer from 'vite-plugin-simple-json-server';
 import GithubActionsReporter from 'vitest-github-actions-reporter';
+
+const logger = createLogger();
+const loggerInfo = logger.info;
+
+// suppress svg build output during CI (spammy due to cryptocurrency-icons)
+logger.info = (msg, options) => {
+	if (!!process.env.CI && /immutable\/assets\/.*\.svg/.test(msg)) return;
+	loggerInfo(msg, options);
+};
 
 export default defineConfig({
 	plugins: [
@@ -25,6 +35,14 @@ export default defineConfig({
 		fs: {
 			allow: [process.cwd()]
 		}
+	},
+
+	// suppress svg build output during CI (spammy due to cryptocurrency-icons)
+	customLogger: logger,
+
+	// suppress plotly build warnings (remove once ploty.js is gone)
+	build: {
+		chunkSizeWarningLimit: 1200
 	},
 
 	// vitest configuration for unit tests (`npm run test:unit`)

